CMakeLists.txt

```diff
@@ -344,6 +344,13 @@ include_directories(${Boost_INCLUDE_DIRS})
 
 find_package(Threads REQUIRED)
 
+option(WITH_SELINUX "build SELinux policy" OFF)
```

Comment:
@smithfarm yes, `option()` is `OFF` by default. and i put `OFF` here to make it explicit.